client/src/components/Captain.tsx

```diff
@@ -0,0 +1,31 @@
+import { Button, DialogActions, DialogContent } from "@mui/material";
+import Chatbot from "./Chatbot";
+import { BootstrapDialog, BootstrapDialogTitle } from "./helpers/SideDialog";
+
+
+export default function Captain() {
+
+  return (
+    <BootstrapDialog
+      // onClose={handleClose}
+      aria-labelledby="customized-dialog-title"
+      open={true}
+    >
+      <BootstrapDialogTitle id="customized-dialog-title"
+        onClose={()=>undefined}
```

Comment:
ha?

Reply:
fix it later, I just want something to show and than I will improve it